### ISSUE TITLE
tapOn textfield not working

### ISSUE THREAD

**Issue by sanskar10100** [mobile-dev-inc/maestro#149]
I have a Jetpack Compose app, which features a textfield in a dialog having the label: "Enter code or url"
While running a Mastro test, the `tapOn` command is not working for this textfield. Please see:

![image](https://user-images.githubusercontent.com/22092047/189314392-ea80d733-3a35-42fd-ae64-3eeda1355236.png)

https://user-images.githubusercontent.com/22092047/189314637-a1aa8e27-1414-4a61-b10d-479b45bde558.mp4

On running `maestro hierarchy`, I can't find the "Enter code or url" label if the textfield is unfocused, but I can see it if it's focused. See:

https://user-images.githubusercontent.com/22092047/189315300-560a6b75-9c2e-4722-8792-8669f844abfe.mp4

Hierarchy outputs:

[after_focus.txt](https://github.com/mobile-dev-inc/maestro/files/9534047/after_focus.txt)
[before_focus.txt](https://github.com/mobile-dev-inc/maestro/files/9534048/before_focus.txt)

**Comment by dmitry-zaitsev** [comment]
Would it be possible for you to also share the APK file with us?

**Comment by sanskar10100** [comment]
Sure, you can grab it here: https://github.com/sanskar10100/Panel/releases/download/v0.2/Panel.0.2.apk

The code in question can be found here: https://github.com/sanskar10100/Panel/blob/805fbf4d8cc30cb9d9f6efab9d9832d0fed55850/app/src/main/java/dev/sanskar/panel/ui/home/HomeFragment.kt#L117

**Comment by sanskar10100** [comment]
Also, @dmitry-zaitsev are there any future plans to include support for specific IME actions in Maestro later? I rely on ImeAction.Done quite heavily in this app, and tried using `pressKey: enter` for the purpose but it didn't work out.

**Comment by dmitry-zaitsev** [comment]
Sorry for the late reply. Based on your code, it does not seem like you are actually setting `Modifier.semantics` anywhere, unless I am missing it?

**Comment by dmitry-zaitsev** [comment]
On IME actions - could you let us know what kind of IME actions you are looking for specifically?

**Comment by sanskar10100** [comment]
@dmitry-zaitsev that's correct, the `Modifier.semantics` code hasn't been pushed to the server yet. It's still on my local. Just wanted to let you know the test is working if I add that. 

On the IME front, I was specifically looking for IME.ACTION_DONE.

**Comment by amanjeetsingh150** [comment]
Hey @sanskar10100 the right way on interacting with Jetpack compose composables in this case text field would be test tags. We have documented this here:

https://maestro.mobile.dev/platform-support/android-jetpack-compose

Let me know if this works for you. Feel free to open an issue if something is missing!

On IME actions if you are still facing the issue can you open a separate issue for it with reproducing steps?
